src/NzbDrone.Core/Configuration/ConfigFileProvider.cs

```diff
@@ -202,14 +206,33 @@ public string UrlBase
         // public string UiFolder => GetValue("UiFolder", "UI", false);GetValue("UiFolder", "UI", false);
         public string UiFolder => "UI";
 
+        public string InstanceName
+        {
+            get
+            {
+                var instanceName = GetValue("InstanceName", BuildInfo.AppName);
+
+                if (instanceName.StartsWith(BuildInfo.AppName) || instanceName.EndsWith(BuildInfo.AppName) )
+                {
+                    return instanceName;
+                }
+                return BuildInfo.AppName;
+            }
+        }
```

Comment:
@markus101 this is to stop it being worked around from not having Sonarr in the InstanceName